docs/book/component-gallery/orchestrators/kubernetes.md

```diff
@@ -63,6 +63,35 @@ You can now run any ZenML pipeline using the Kubernetes orchestrator:
 python file_that_runs_a_zenml_pipeline.py
 ```
 
+For additional configuration of the Kubernetes orchestrator, you can pass
+`KubernetesOrchestratorSettings` which allows you to configure the following attributes:
+
+* `node_affinity`: NodeAffinity match expressions.
+* `tolerations`: Toleration settings for the pod.
```

Comment:
This could need a bit more detail on what those attributes actually do and why/when you would want to configure them.